Gems/PhysX/Code/Source/EditorColliderComponent.h

```diff
@@ -145,7 +145,8 @@ namespace PhysX
         void OnDeselected() override;
 
         // DisplayCallback
```

Comment:
nit: Could we update this to `DebugDraw::DisplayCallback overrides ...` and add a line break after to separate from the functions that are not overrides. Thanks!